### PR TITLE
Button Text Change Done

### PR DESCRIPTION
**Pull request by rajcrk**
Fixes #[Add issue number here.]

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts
* [ ] PR is descriptively titled
* [ ] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

Thanks!

### CODE REVIEW

**Comment by HarshKhandeparkar** [comment]
@rajcrk please remove

>#[Add issue number here.]

and please swap it with the issue link.

> Fixes http://link-to-the-issue

**Comment by HarshKhandeparkar** [comment]
Also it would be cool if you change the text besides the button which reads "press save to see changes".

File: examples/lib/defaultHtmlStepUi.js [hunk 1]
```diff
@@ -130,7 +130,7 @@ function DefaultHtmlStepUi(_sequencer, options) {
       $(step.ui.querySelectorAll(".target")).on('change',toggleSaveButton);
 
       $(step.ui.querySelector("div.details")).append(
-        "<p><button class='btn btn-default btn-save' disabled = 'true' >Save</button><span> Press save to see changes</span></p>"
+        "<p><button class='btn btn-default btn-save' disabled = 'true' >Apply</button><span> Press save to see changes</span></p>"
```
Comment (HarshKhandeparkar) [inline examples/lib/defaultHtmlStepUi.js]: You can change the text besides the button. That will look better.

**Comment by HarshKhandeparkar** [comment]
Great work by the way. You'll have to wait until @publiclab/reviewers review it.

**Comment by Rishabh570** [comment]
Hi @rajcrk, can you please change the span text as suggested? This is good to go after that...

**Comment by gauravano** [comment]
Hi @rajcrk, just checking if you need any help. Thanks!

**Comment by HarshKhandeparkar** [comment]
These changes have been applied already. Maybe in some other PR. Can this one be closed? @jywarren @gauravano

**Comment by gauravano** [comment]
Thanks @HarshKhandeparkar for the status update. Closing this PR as `master` contains the required chages and some work is tracked in #517. Thanks @rajcrk for working on it!

**Pull request closed without merging**